src/aiidalab_qe/app/result/components/viewer/structure/structure.py

```diff
@@ -42,6 +54,19 @@ def _render(self):
         ngl._set_size("100%", "300px")
         ngl.control.zoom(0.0)
 
+    def _initialize_structure_info(self, structure):
+        self.structure_info = ipw.HTML(
+            f"""
+            <div style='line-height: 1.4;'>
+                <strong>PK:</strong> {structure.pk}<br>
+                <strong>Label:</strong> {structure.label}<br>
+                <strong>Description:</strong> {structure.description}<br>
+                <strong>Number of atoms:</strong> {len(structure.sites)}<br>
+                <strong>Creation time:</strong> {format_time(structure.ctime)} ({relative_time(structure.ctime)})<br>
+            </div>
+            """
+        )
```

Comment:
```suggestion
    def _get_structure_info(self, structure):
        return ipw.HTML(
            f"""
            <div style='line-height: 1.4;'>
                <strong>PK:</strong> {structure.pk}<br>
                <strong>Label:</strong> {structure.label}<br>
                <strong>Description:</strong> {structure.description}<br>
                <strong>Number of atoms:</strong> {len(structure.sites)}<br>
                <strong>Creation time:</strong> {format_time(structure.ctime)} ({relative_time(structure.ctime)})<br>
            </div>
            """
        )
```

If you agree with the previous comment